src/errors/errors.js

```diff
@@ -0,0 +1,20 @@
+import { logError } from '../middleware/logging';
+
+export const errorMessages = {
+  HealthRecordNotFound: 'No complete health record was found with given criteria',
+  MessageNotFound: 'There were no undeleted messages associated with conversation id',
```

Comment:
May be better instead of "undeleted", maybe term it "There are no existing messages associated with conversation id" ?

Reply:
Thanks, changed to `no existing messages`